app/cdap/components/Administration/AdminTabSwitch/index.js

```diff
@@ -57,16 +63,26 @@ export default class AdminTabSwitch extends Component {
     });
   }
 
-  renderTabTitle(isManagement = true) {
+  renderTabTitle(activeTab) {
+    const { MANAGEMENT, CONFIGURATION, TETHERING } = TAB_LABELS;
+
     return (
       <span className="tab-title">
-        <h5 className={classnames({ active: isManagement })}>
+        <h5 className={classnames({ active: activeTab === MANAGEMENT })}>
           <Link to="/administration">{T.translate(`${PREFIX}.Tabs.management`)}</Link>
         </h5>
         <span className="divider"> | </span>
-        <h5 className={classnames({ active: !isManagement })}>
+        <h5 className={classnames({ active: activeTab === CONFIGURATION })}>
           <Link to="/administration/configuration">{T.translate(`${PREFIX}.Tabs.config`)}</Link>
         </h5>
+        {Theme.tethering && (
+          <React.Fragment>
```

Comment:
js files don't like the shorthand for <React.Fragment>

Reply:
switch this to a typescript file then :)